tests/foundry/client-esm/applications/ui/context.test-d.ts

```diff
@@ -14,20 +12,57 @@ const menuA = foundry.applications.ui.ContextMenu.create(testAppV2, testAppV2.el
   jQuery: false,
   fixed: true,
 });
-expectTypeOf(menuA.fixed).toEqualTypeOf<true>();
+expectTypeOf(menuA.fixed).toBeBoolean();
 expectTypeOf(menuA.menuItems[0]?.callback(htmlElement)).toBeVoid();
 // @ts-expect-error Callback for jQuery: false takes an HTML Element
-expectTypeOf(menuA.menuItems[0]?.callback(jquery)).toBeVoid();
+menuA.menuItems[0]?.callback(jquery);
+
+// Need to pass explicit jQuery in options
+foundry.applications.ui.ContextMenu.create(testAppV2, testAppV2.element, ".foobar", {
```

Comment:
Can you add some comments saying "should be deprecated" to explain the existence of these lines? It also will help if we switch to `attest` AND `attest` adds an `isDeprecated`.